framework_tvb/tvb/core/services/project_service.py

```diff
@@ -322,13 +322,13 @@ def remove_project(self, project_id):
             for burst in project_bursts:
                 dao.remove_entity(burst.__class__, burst.id)
 
-            project_folder = self.structure_helper.get_project_folder(project2delete)
-            self.structure_helper.remove_project_structure(project2delete.name)
-            encrypted_path = encryption_handler.compute_encrypted_folder_path(project_folder)
+            project_folder = self.storage_interface.get_project_folder(project2delete.name)
```

Comment:
Same here, the code handling the removal of the project dir and the encryption sync could move under `StorageInterface`.